aclmapping/bank/mappings.go

```diff
@@ -81,6 +81,17 @@ func MsgSendDependencyGenerator(keeper aclkeeper.Keeper, ctx sdk.Context, msg sd
 			IdentifierTemplate: hex.EncodeToString(authtypes.CreateAddressStoreKeyFromBech32(msgSend.FromAddress)),
 		},
 
+		{
+			AccessType:         sdkacltypes.AccessType_READ,
+			ResourceType:       sdkacltypes.ResourceType_KV_AUTH_GLOBAL_ACCOUNT_NUMBER,
+			IdentifierTemplate: hex.EncodeToString(authtypes.GlobalAccountNumberKey),
+		},
+		{
+			AccessType:         sdkacltypes.AccessType_WRITE,
```

Comment:
doesn't this mean that NO bank sends can be parallelized? Is there some check we can do before including this dependency so that we can retain some level of performance in the normal case for bank sends?